### PR TITLE
Icon proposal for OHdio

### PR DESCRIPTION
**Pull request by service-paradis**
# Description

Icon proposal for OHdio (https://play.google.com/store/apps/details?id=rc.appradio.android)

### Added
OHdio (`rc.appradio.android`)

## Contributor's checklist

- [x] I made the icons following [the Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md) (upd. Jan 2024) and will make changes if someone suggests. I will also make sure that Lawnicons builds correctly.

### CODE REVIEW

File: svgs/ohdio.svg
Comment (x9136) [inline svgs/ohdio.svg:1]: It's worth to make the long side closer to 160px (160.48px now).

Comment (service-paradis) [reply svgs/ohdio.svg:1]: Sure! Should be good to go now!

**Review by x9136 — APPROVED** [review]

**Comment by service-paradis** [comment]
@x9136 Do you want me to remove the typos corrections that I made in order to merge this one? I just wanted to fix the failed actions caused by this.

**Comment by x9136** [comment]
@service-paradis The choice is yours. I am fine with both waiting for the code review and leaving only the icon in this PR.

**Comment by x9136** [comment]
Approved by SuperDragonXD in our Discord.

**Comment by hadiphn** [comment]
good

**Pull request merged**